ci-operator/config/open-cluster-management/must-gather/open-cluster-management-must-gather-main.yaml

```diff
@@ -0,0 +1,30 @@
+base_images:
+  base:
+    name: "4.6"
```

Comment:
Are you sure 4.6 is the right version to use for testing? Latest stable is 4.7.

Reply:
I think most of our images use 4.6, and that is the rationale for using 4.6 instead of 4.7. @jlpadilla right?

Reply:
Yes, I'm updating our repo to use the latest 4.7, then I'll update this PR to match (4.7)